projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/banking/loan.ts

```diff
@@ -94,7 +94,7 @@ export function checkMiniCart() {
     products: [
       {
         title: ' Start Date: ',
-        value: ' 12 Dec 2021 ',
+        value: ' 12 Dec 2023 ',
```

Comment:
Are these spaces before and after the string left intentionally? Seems redundant.

Reply:
Fixed